### PR TITLE
Readme docs updates: pip, composer & sudo build.sh

### PR DESCRIPTION
**Pull request by JasonHoku**
pip --> pip3 for more global install stability.

 composer docs added for clarity 

sudo build.sh ( this is necessary with the default docker setup )

:)

### CODE REVIEW

**Comment by JasonHoku** [comment]
Any idea why my VSCodium edit of this would yield these result?

File: README.md [hunk 1]
```diff
@@ -48,31 +48,34 @@ modern stack.
...
-  - [Utility Functions](#utility-functions)
-    - [login](#login)
-    - [logout](#logout)
-    - [isAuthenticated](#isauthenticated)
-  - [Routes](#routes)
-  - [Higher Order Components](#higher-order-components)
-    - [PrivateRoute](#privateroute)
-- [Deployment](#deployment)
-- [Contributing](#contributing)
+- [FastAPI + React · !build [![license](https://img.shields.io/github/license/peaceiris/actions-gh-pages.svg)](LICENSE) [![Dependabot Status](https://img.shields.io/badge/Dependabot-active-brightgreen.svg)](https://dependabot.com)](#fastapi--react--img-srchttpsgithubcombuuntufastapi-reactworkflowsbuildbadgesvg-altbuild-img-srchttpsimgshieldsiogithublicensepeaceirisactions-gh-pagessvg-altlicense-img-srchttpsimgshieldsiobadgedependabot-active-brightgreensvg-altdependabot-status)
```
Comment (Buuntu) [inline README.md:51]: This, features, and table of contents should not be a part of the TOC.

File: README.md [hunk 2]
```diff
@@ -122,11 +132,11 @@ You will need to put in a few variables and it will create a project directory
 
 ## Develop
 
-Change into your project directory and run:
+Change into your project directory and run as sudo priv (required for docker):
```
Comment (Buuntu) [inline README.md:135]: Run with elevated privileges is better.

File: README.md [hunk 3]
```diff
@@ -122,11 +132,11 @@ You will need to put in a few variables and it will create a project directory
...
 
 ```bash
-chmod +x scripts/build.sh
-./scripts/build.sh
+sudo chmod +x scripts/build.sh
```
Comment (Buuntu) [inline README.md:138]: I don't think you need sudo for this, you might for docker-compose is all.

**Comment by JasonHoku** [comment]
Okay, I have corrected the strange formatting caused a MD formatted for VS Code and removed the sudo for the build.sh

Still got the prettier failed error, is that okay?

Again, thank you so much for working with me on this.

**Comment by Buuntu** [comment]
> Okay, I have corrected the strange formatting caused a MD formatter for VS Code and removed the sudo for the build.sh
> 
> Still got the prettier failed error, is that okay?
> 
> Again, thank you so much for working with me on this.

I think you need to setup Prettier in VSCode, it may not be running on the README.

**Pull request merged**